include/auth_ejabberd.php

```diff
@@ -320,7 +324,7 @@ private function writeLog($sMessage) {
 	 * @param string $sMessage The logfile message
 	 */
 	private function writeDebugLog($sMessage) {
-		if ($this->bDebug)
+		if ($this->bDebug) {
```

Comment:
I don't mind running `php util\typo.php` each time @Quix0r fixes the last syntax error, it's quick and painless for me. If it bothers him, he'll start doing it on his own after a while. 🙂